core/src/wallet/tezos/explorers/ExternalTezosLikeBlockchainExplorer.cpp

```diff
@@ -86,7 +86,11 @@ namespace ledger {
                         } else if (fees.find('.') != std::string::npos) {
                             fees = api::BigInt::fromDecimalString(fees, 6, ".")->toString(10);
                         }
-                        return std::make_shared<BigInt>(fees);
+                        // Since nodes are giving some awkward values, we set a threshold to avoid having really fees
+                        // Factor for threshold is inspired from other XTZ wallets
+                        return std::stoi(fees) > 6 * std::stoi(api::TezosConfigurationDefaults::TEZOS_DEFAULT_FEES) ?
```

Comment:
Can you stored the calculation `6 * std::atoi(api::TezosConfigurationDefaults::TEZOS_DEFAULT_FEES)` to avoid unwanted computation and to give a name to that magic value ?

Reply:
i kinda agree, to me this should be a constant as well. because the DEFAULT_FEES is already arbitrary and it would be unwanted that a change of default fees affect the "max cap". because the minimum cap can vary independently of the max cap